package.json

```diff
@@ -71,7 +71,7 @@
     "yaml": "^2.2.2",
     "tough-cookie": "^4.1.3",
     "semver": "^7.5.2",
-    "@cypress/request": "^3.0.0",
+    "@cypress/request": "^3.0.9",
```

Comment:
can we remove this and just pull from upstream?

Reply:
Removed it since we don't actually use requests, just updated yarn.lock